RELEASE_NOTE.md

```diff
@@ -1,5 +1,33 @@
 # OpenPAI Release Note
 
+## Jan 2021 (version 1.4.0)
+
+- Improve Web Portal Experience 
+  - Fix Home page overlap issue #5213 #5180
+  - Add filter, search box and export csv button in task detail list #5175
+  - Create a new page for yaml editor #5172
+
+- Marketplace related update
+  - Please see https://github.com/microsoft/openpaimarketplace/issues/152 for more details
+
+- Support different types of computing hardware #5138 
+
+- Deployment process refinement 
```

Comment:
We can highlight: 1. add support for cpu-only worker installation 2. add support for heterogeneous workers

Reply:
added

Reply:
The two items should be under "Deployment process refinement", not "Support different types of computing hardware"

Reply:
done